### ISSUE TITLE
[1.18.2] Server crash on start

### ISSUE THREAD

**Issue by Shadowrs** [Lightman314/Lightmans-Discord-Integration#13]
lightmansdiscord-1.18-0.1.0.1-server
ATM7 modpack (200+ mods)
Forge 40.2.0
MC 1.18.2

```
[14:47:17] [main/INFO] [ne.mi.fm.lo.mo.JarInJarDependencyLocator/]: Found 28 dependencies adding them to mods collection
Exception in thread "main" java.lang.module.ResolutionException: Modules lightmansdiscord and com.fasterxml.jackson.databind export package com.fasterxml.jackson.databind to module mob_grinding_utils
        at java.base/java.lang.module.Resolver.resolveFail(Resolver.java:901)
        at java.base/java.lang.module.Resolver.failTwoSuppliers(Resolver.java:815)
        at java.base/java.lang.module.Resolver.checkExportSuppliers(Resolver.java:736)
        at java.base/java.lang.module.Resolver.finish(Resolver.java:380)
        at java.base/java.lang.module.Configuration.<init>(Configuration.java:140)
        at java.base/java.lang.module.Configuration.resolveAndBind(Configuration.java:494)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.ModuleLayerHandler.buildLayer(ModuleLayerHandler.java:70)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.TransformationServicesHandler.buildTransformingClassLoader(TransformationServicesHandler.java:60)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.Launcher.run(Launcher.java:104)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.Launcher.main(Launcher.java:77)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:26)
        at MC-BOOTSTRAP/cpw.mods.modlauncher@9.1.3/cpw.mods.modlauncher.BootstrapLaunchConsumer.accept(BootstrapLaunchConsumer.java:23)
        at cpw.mods.bootstraplauncher@1.0.0/cpw.mods.bootstraplauncher.BootstrapLauncher.main(BootstrapLauncher.java:149)

```

**Comment by Lightman314** [comment]
Please attach the crash report file, instead of just pasting a small portion of the crash.

**Comment by Shadowrs** [comment]
[Latest.log ](https://pastebin.com/T2meWrXt)
[debug.log ](https://gist.github.com/Shadowrs/a3dcfedcec578406b483142eb75b37eb)
there is no crash-report file
looks like a java module system error - maybe a forge loader issue with these two mods using java9 module system for deps? `Exception in thread "main" java.lang.module.ResolutionException: Modules lightmansdiscord and com.fasterxml.jackson.databind export package com.fasterxml.jackson.databind to module mob_grinding_utils`

**Comment by Lightman314** [comment]
Should be fixed in LDI v0.2.0.1 as it now uses proper Jar-in-Jar libraries.
